src/language/__tests__/printer-test.js

```diff
@@ -174,4 +178,45 @@ describe('Printer: Query document', () => {
     `),
     );
   });
+
+  it('keeps arguments on one line if line is short (<= 80 chars)', () => {
```

Comment:
The kitchen sink test should be the last, please move these tests before it.

Reply:
✅